CMakeLists.txt

```diff
@@ -950,7 +950,9 @@ endif()
 
 foreach(ZLIB_INSTALL_LIBRARY ${ZLIB_INSTALL_LIBRARIES})
     target_include_directories(${ZLIB_INSTALL_LIBRARY} PUBLIC
-        ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
+        "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR};${CMAKE_CURRENT_SOURCE_DIR}>"
+        "$<INSTALL_INTERFACE:include>"
+        )
```

Comment:
I would prefer if closing brace wasn't on a separate line. That formatting is good if there is like a list of files and that list may change a lot but I don't believe this instance is that.

Reply:
Done.